frontend/src/features/modules/ModulesPage.tsx

```diff
@@ -1,21 +1,34 @@
-import { useMemo } from 'react'
+import type React from 'react'
+import { useMemo, useState } from 'react'
 import { useSchema } from '../../api/schema/use-schema'
+import { ResizableHorizontalPanels } from '../../components/ResizableHorizontalPanels'
 import { ModulesTree } from './ModulesTree'
 import { moduleTreeFromSchema } from './module.utils'
 
-export const ModulesPage = () => {
+export const ModulesPanel = () => {
+  return (
+    <div className='flex-1 py-2 px-4'>
+      <p>Content</p>
+    </div>
+  )
+}
+
+export const ModulesPage = ({ body }: { body: React.ReactNode }) => {
   const schema = useSchema()
   const tree = useMemo(() => moduleTreeFromSchema(schema?.data || []), [schema?.data])
+  const [treeWidth, setTreeWidth] = useState(Number(localStorage.getItem('tree_w')) || 300)
```

Comment:
nit: but we could make this a `const` so we don't typo in the future 😂 

Also, maybe call this `tree_width` just to be more clear?

Reply:
whoops how embarassing, thanks for reminding me! 😂